latest/index.bs

```diff
@@ -331,24 +317,917 @@ Conforming readers:
 - MAY ignore other groups or arrays under the root of the hierarchy.
 
 
+"coordinateSystems" metadata {#coord-sys-md}
+--------------------------
+
+A "coordinate system" is a collection of "axes" / dimensions with a name. Every coordinate system:
+- MUST contain the field "name", that gives the space name. The values MUST be non-empty and unique.
+- MUST contain the field "axes", whose value is a valid set of "axes" (see below).
+
+<div class=example>
+
+```json
+{
+    "name" : "volume_micrometers",
+    "axes" : [
+        {"name": "x", "type": "space", "unit": "micrometer"},
+        {"name": "y", "type": "space", "unit": "micrometer"},
+        {"name": "z", "type": "space", "unit": "micrometer"}
+    ]
+}
+```
+</div>
+
+The order of the `"axes"` list matters and defines the index of each array dimension and coordinates for points in that
+coordinate system. For the above example, the `"x"` dimension is the first dimension. The "dimensionality" of a coordinate system
+is indicated by the length of its "axes" array.  The "volume_micrometers" example above is three dimensional (3D).
+
+### "axes" metadata {#axes-md}
+
+"axes" describes the dimensions of a coordinate systems. It is a list of dictionaries, where each dictionary describes a dimension (axis) and:
+- MUST contain the field "name" that gives the name for this dimension. The values MUST be unique across all "name" fields.
+- SHOULD contain the field "type". It SHOULD be one of "array", "space", "time", "channel", "coordinate", or "displacement" but MAY take other values for custom axis types that are not part of this specification yet.
+- MAY contain the field "discrete". The value MUST be a boolean, and is `true` if the axis represents a discrete dimension.
```

Comment:
Maybe some more elaboration of what a "discrete axis" means here. I would look to the [cf conventions](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.8/cf-conventions.html#discrete-axis) treatment on the subject as an example.

Reply:
see https://github.com/bogovicj/ngff/blob/coord-transforms/latest/index.bs#L431-L462

though some of that feels a bit like instruction / tutorial (?) and not sure it quite belongs. we'll see.

Reply:
Is there a fixed relation with `type`? For example I imagine that `channel` would always be discrete. Just a thought, I am not sure if we want to capture this in the spec. (Ok, this is mentioned below.)

Reply:
Also: I think `discrete:false` implies that an axis is interpolatable and vice versa. We should explicitly state this here.